packages/frontend-v2/pages/index.tsx

```diff
@@ -25,25 +30,28 @@ interface InfoTextProps {
 }
 
 const Home: NextPage = () => {
+  const router = useRouter();
   return (
     <Box>
-      <Header />
-      <Banner />
+      <Header router={router} />
+      <Banner router={router} />
       <Info />
     </Box>
   );
 };
 
-const Header = (): JSX.Element => {
+const Header = ({ router }: RouterProps): JSX.Element => {
```

Comment:
in this situation (see next comment) you don't need router. however if you did, you can just access the router directly in this component, it doesn't need to be a prop